src/sidebar/components/selection-tabs.js

```diff
@@ -29,6 +29,9 @@ function Tab({
       })}
       onMouseDown={onChangeTab.bind(this, type)}
       onTouchStart={onChangeTab.bind(this, type)}
+      role="tab"
+      tabIndex="0"
+      ariaSelected={selected}
```

Comment:
Screen reader is not picking this up, I think it needs to be `aria-selected`

Reply:
Ah, interesting. It turns out this works in the latest Safari because elements have an `ariaSelected` property, but that is not yet implemented in Chrome. Setting `aria-selected` indeed works in all browsers as it sets the `aria-selected` attribute directly.